### PR TITLE
Add arguments to orb

### PR DESCRIPTION
**Pull request by andreaTP**
Add additional arguments for the coverage script.

Fix:
https://github.com/codacy/codacy-meta/issues/345

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -207,17 +221,17 @@ workflows:
...
-              sbt test
-            else
-              sbt clean coverage test coverageReport
-              sbt coverageAggregate
-              bash <(curl -Ls https://coverage.codacy.com/get.sh) report --skip
-              sbt "it:test"
-            fi
+            sbt clean coverage test coverageReport
+            sbt coverageAggregate
+            # bash <(curl -Ls https://coverage.codacy.com/get.sh) report --skip
```
Comment (josemiguelmelo) [inline .circleci/config.yml:226]: We could remove this line, right?

Comment (andreaTP) [reply .circleci/config.yml]: done

File: .circleci/config.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ version: 2.1
 
 orbs:
   codacy: codacy/base@2.12.1
+  coverage-reporter: codacy/coverage-reporter@dev:7.9.1-addparamstoorb.abf6e09
```
Comment (andreaTP) [inline .circleci/config.yml:5]: I'll follow up with a PR for having a stable orb here as soon as we publish with this PR merged.

**Comment by franciscodua** [comment]
Superseded by #208

**Pull request closed without merging**